scenario/LiquidationBotScenario.ts

```diff
@@ -95,13 +95,14 @@ for (let i = 0; i < MAX_ASSETS; i++) {
       ],
     }
   };
+  // XXX enable for mainnet WETH
   scenario(
     `LiquidationBot > liquidates an underwater position of $asset${i} with no maxAmountToPurchase`,
     {
       upgrade: {
         targetReserves: exp(20_000, 18)
       },
-      filter: async (ctx) => await isValidAssetIndex(ctx, i) && matchesDeployment(ctx, [{network: 'mainnet'}, {network: 'polygon'}]),
+      filter: async (ctx) => await isValidAssetIndex(ctx, i) && matchesDeployment(ctx, [{network: 'mainnet', deployment: 'usdc'}, {network: 'polygon'}]),
```

Comment:
Mainnet WETH stalls for these liquidation bot scenarios because it cannot source that much base asset. I'm skipping these for now because they are not related to this PR.